src/NovelRT/Timing/Timestamp.cpp

```diff
@@ -0,0 +1,7 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#include <NovelRT.h>
```

Comment:
File can be deleted?

Reply:
Uhhh...I already deleted this. Why is it still here? Let me double check.

Reply:
Ok, somehow this re-added itself. Thanks Git. 😒